charts/ztka/Chart.yaml

```diff
@@ -25,5 +25,5 @@ dependencies:
     condition: deploy.contour.enable
 maintainers:
   - name: Paralus Team
-version: "0.1.3"
+version: "0.1.4"
 appVersion: "v0.1.3"
```

Comment:
Do we want to bump appVersion too?

Reply:
Should we bump it up? The idea currently is to keep it in sync with `paralus/paralus`.

Reply:
I know we decided to keep it same but there is no change in app so I guess it could deviate and that's the reason we have two versions right.

Reply:
Yup

Reply:
Make sense. This release has only helm chart change. So no need to update appVersion.